README.md

```diff
@@ -35,10 +35,20 @@ Although the Windows Package Manager YAML Generator can create YAML files with m
 
 ## Test your manifest
 Now that you have authored your manifest, you should make sure it works as expected.
+
+### Locally
 1) Verify the syntax.  You can do that by typing the following command: `winget validate <manifest>`
 2) Test the install.  You can do that by installing the manifest: `winget install -m <manifest>`
 For more details, see [packages](https://docs.microsoft.com/windows/package-manager/package).
 
+### In Windows Sandbox
+You can use the [`Tools\SandboxTest.ps1`](Tools/SandboxTest.ps1) script for testing a manifest installation in [Windows Sandbox](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-sandbox/windows-sandbox-overview). The manifest will be also validated.
+
+Just provide the path to manifest as parameter:
+```powershell
+.\Tools\SandboxTest.ps1 <path-to-manifest>
```

Comment:
@palenshus Well, my goal was to support the two kinds of manifests to keep backwards compatibility (it was files previously). Since you said you're not from the winget-cli team, it may be better to ask them about it. @denelon, any suggestion?

If yet we decide to keep accepting both files and folders, the help can be surely enhanced to clarify that.

Reply:
@palenshus Well, my goal was to support the two kinds of manifests to keep backwards compatibility (it was files previously). Since you said you're not from the winget-cli team, it may be better to ask them about it. @denelon, any suggestion?

If yet we decide to keep accepting both files and folders, the help can be surely enhanced to clarify that.

Reply:
If you have a singleton manifest you can point to the file, but the directory option works for both cases.

Reply:
I see, I was only trying it on multi-file schemas. I'd just make it take a directory in both cases for consistency. Winget will validate the directory and ensure that if the manifestType is singleton, then there's only the one file in there